### PR TITLE
Update collaborating-on-grants.md

### PR DESCRIPTION
**Pull request by kariljordan**
Added 2023 org priorities. I set myself a recurring task to add the priorities each year when they're developed.

### CODE REVIEW

**Comment by maneesha** [comment]
A note that this PR says it is intended to add in 2023 priorities. However, the changes I see are all minor changes to language/wording.

File: topic_folders/fundraising/collaborating-on-grants.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
-2. **Request to meet one of our program leads**. If you have an idea that will support one of the Carpentries' existing priorities and would like to co-write a grant proposal, please schedule a meeting with the corresponding program lead, using the links below. [Minimum requirements](https://docs.google.com/document/d/1-08lXysNEDc0lka0UB6cwToTq7XKi79lgZVMQhOcGZo/edit#heading=h.ucwcxk7d18gv) for co-writing a grant proposal with The Carpentries are provided below.
+2. **Request to meet one of our program leads**. If you have an idea supporting one of the Carpentries' priorities (i.e., capital campaign, community health, process improvement) and would like to co-write a grant proposal, please schedule a meeting with the corresponding program lead using the links below. [Minimum requirements](https://docs.google.com/document/d/1-08lXysNEDc0lka0UB6cwToTq7XKi79lgZVMQhOcGZo/edit#heading=h.ucwcxk7d18gv) for co-writing a grant proposal with The Carpentries are provided below.
```
Comment (kariljordan) [inline topic_folders/fundraising/collaborating-on-grants.md:8]: @maneesha the priorities were added in the parentheses.

File: topic_folders/fundraising/collaborating-on-grants.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
 # A Guide for Collaborating with The Carpentries on Grants
 
 ## Statement to Interested Collaborators
-This guide was developed for potential collaborators interested in involving The Carpentries in a grant proposal. We value all contributions to our mission and are excited to collaborate with individuals and organisations interested in mission-aligned activities (i.e. Carpentries workshops, instructor training, curriculum development). We offer two options for collaborating with The Carpentries on grant proposals:
+This guide was developed for potential collaborators interested in involving The Carpentries in a grant proposal. We value all contributions to our mission and are excited to collaborate with individuals and organisations interested in mission-aligned activities (i.e., Carpentries workshops, instructor training, curriculum development). We offer two options for collaborating with The Carpentries on grant proposals:
```
Comment (acrall) [inline topic_folders/fundraising/collaborating-on-grants.md:4]: Our style guide states to not include a comma in this case: https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#i-e

File: topic_folders/fundraising/collaborating-on-grants.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
-1. **Request a letter of support**. If you are planning to budget Carpentries programming (i.e. workshops, membership, instructor training seats) into your grant proposal and would like to request a letter of support, please [complete this form](https://carpentries.typeform.com/to/oK25vSPE). Pricing for Carpentries programming is located on our [membership page](https://carpentries.org/membership/). All requests for letters of support require **five business days** to be fulfilled.
+1. **Request a letter of support**. If you are planning to budget Carpentries programming (i.e., workshops, membership, instructor training seats) into your grant proposal and would like to request a letter of support, please [complete this form](https://carpentries.typeform.com/to/oK25vSPE). Pricing for Carpentries programming is on our [membership page](https://carpentries.org/membership/). All requests for letters of support require **five business days** to be fulfilled.
```
Comment (acrall) [inline topic_folders/fundraising/collaborating-on-grants.md:6]: Our style guide states to not include a comma in this case: https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#i-e

File: topic_folders/fundraising/collaborating-on-grants.md (hunk 1 above)
Comment (acrall) [reply topic_folders/fundraising/collaborating-on-grants.md:8]: @kariljordan Do we have a description we can point to for the priority areas? To provide more context on what is meant by capital campaign, community health.....

File: topic_folders/fundraising/collaborating-on-grants.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
   - [Curriculum](https://calendly.com/tobyhodges) - Toby Hodges
   - [Instructor Training](https://calendly.com/karenword) - Karen Word
   - [Membership](https://calendly.com/ebecker-1) - Erin Becker
   - [Workshops](https://calendly.com/sheraaron/) - SherAaron Hurt
 
-We invite you to include The Carpentries in your grant to support workshops, membership, and instructor training whether you apply as an individual institution or as a consortium. We are also happy to collaborate on grant proposals that support our [mission](https://carpentries.org/about/), [core values](https://carpentries.org/values/), and current priorities. This guide should be used as a resource in advance of proposal development.
+We'd like to invite you to include The Carpentries in your grant to support workshops, membership, and instructor training, whether you apply as an individual institution or as a consortium. We are also happy to collaborate on grant proposals that support our [mission](https://carpentries.org/about/), [core values](https://carpentries.org/values/), and current priorities. This guide should be used as a resource in advance of proposal development.
```
Comment (acrall) [inline topic_folders/fundraising/collaborating-on-grants.md:16]:
```suggestion
We would like to invite you to include The Carpentries in your grant to support workshops, membership, and instructor training, whether you apply as an individual institution or as a consortium. We are also happy to collaborate on grant proposals that support our [mission](https://carpentries.org/about/), [core values](https://carpentries.org/values/), and current priorities. This guide should be used as a resource in advance of proposal development.
```

**Review by acrall — APPROVED** [review]
Made some minor recommendations based on our style guide

File: topic_folders/fundraising/collaborating-on-grants.md (hunk 1 above)
Comment (kariljordan) [reply topic_folders/fundraising/collaborating-on-grants.md:8]: - Capital Campaign: Concerted effort to generate unrestricted revenue.
- Community Health: Concerted effort to recognize, empower, and motivate community members.
- Process Improvement: Concerted effort to improve all operational and programmatic workflows; improve and standardise onboarding and offboarding processes for all community roles and segments; and implement automation to reduce administrative time on routine tasks.

**Comment by maneesha** [comment]
@acrall  you noted: 

> Made some minor recommendations based on our style guide

but I don't see any commits beyond Kari's original commit.  Did I miss anything?

**Comment by acrall** [comment]
@maneesha I removed the comma from i.e. in several places. https://docs.carpentries.org/topic_folders/communications/resources/style-guide.html#i-e

**Pull request merged**